test/integration/telemetry/pages/warning.skip

```diff
@@ -1,8 +1,8 @@
 function a(v) {
   return v
 }
-;['index.js'].forEach(f => {
-  require(a('./dynamic-file-imports/' + f))
+;['index.js'].forEach(async f => {
+  await import(a('./dynamic-file-imports/' + f))
```

Comment:
I modified this because for this module is ESM and cjs code caused a failure of telemetry test case